Posts/2021/03/DarwinJS-Contribution-LightningFastandEasyProvisioningofGitwithSSHKeyAuthenticationonWindows.md

```diff
@@ -0,0 +1,117 @@
+---
+post_title: Lightning Fast and Easy Provisioning of Git with SSH Key Authentication on Windows
+username: DarwinJS@sanoys.com
+Catagories: PowerShell
+tags: Git, DevOps
+Summary: Getting your team setup with Git over SSH as quickly as possible!
+featured_image: ./media/DarwinJS-Contribution-LightningFastandEasyProvisioningofGitwithSSHKeyAuthenticationonWindows/windows-git-ssh.png
```

Comment:
I think this will work for the featured image. We may need to scale the size but I won't be able to tell that until it gets copied to WordPress.